R/dl_build_src_gran.R

```diff
@@ -6,7 +6,7 @@
 #' 
 #' @import httr devtools
 #' @export
-dl_build_src <- function(GRAN.dir = '../GRAN'){
+dl_build_src <- function(GRAN.dir = './GRAN'){
```

Comment:
@jread-usgs Do you know why this function had 2 dots, and the binary has 1? Just at straws here